test/removeNPMAbsolutePaths.js

```diff
@@ -406,5 +403,34 @@ describe('removeNPMAbsolutePaths.js', function () {
           expect(writeFile).to.have.been.calledOnce.and.calledWith(filePath);
         });
     });
+
+    it('checks that value of file contents do not contain specific removed fields', function () {
+      writeFile.restore();
+      writeFile = sinon.stub(fs, 'writeFile').callsFake((path, fileContents, cb) => {
```

Comment:
The tests should always be:
```
Configure
Test
Assert
```

To do checks in the arguments send to stub you can use `writeFile.getCall(0).args[1]` to get the argument (not sure if the index should be 1 or other).
So, don't change the `writeFile` stub and just add bellow the line `expect(writeFile).to.have.been.calledOnce.and.calledWith(filePath)`:
```
        const packageJson = JSON.parse(writeFile.getCall(0).args[1]);
        expect(packageJson).to.not.have.property('_inBundle');
        expect(packageJson).to.not.have.property('_where');
        expect(packageJson).to.have.property('_from');
```